src/interfaces.ts

```diff
@@ -83,6 +83,7 @@ export enum ObjectType {
   EportalProduct = 'urn:forlagshuset:object:eportal:product',
   PortfolioCourse = 'urn:forlagshuset:object:portfolio:course',
   PortfolioGroup = 'urn:forlagshuset:object:portfolio:group',
+  ErudioAccess = 'urn:forlagshuset:object:erudio:access', 
```

Comment:
@wrola `ErudioAccess` is very gneric what this entity suppose to do?

Reply:
Change into ` urn:forlagshuset:object:erudio:role-assignment`